project/Dependencies.scala

```diff
@@ -16,6 +16,14 @@ object Dependencies {
   val CouchbaseVersion = "2.7.2"
   val CouchbaseVersionForDocs = "2.7"
 
+  val Silencer = {
```

Comment:
Please add a note and link to explain what this compiler plugin provides.

Reply:
Alright. Note added.